react/Textarea/Textarea.js

```diff
@@ -141,35 +164,75 @@ export default class Textarea extends Component {
         <div
           data-automation="backdrop"
           ref={this.storeTextareaBackdropRef}
-          className={classnames(styles.textarea, styles.backdrop)}>
+          className={classnames(styles.textarea, styles.backdrop)}
+        >
           {formattedText}
         </div>
         {renderTextarea({ onScroll: this.onScroll }, styles.highlightTextarea)}
       </div>
-    ) : renderTextarea();
+    ) : (
+      renderTextarea()
+    );
   }
 
   render() {
     const { id, className, valid, tone } = this.props;
     const classNames = classnames({
       [styles.root]: true,
-      [styles.invalid]: typeof tone !== 'undefined' ? tone === TONE.CRITICAL : valid === false,
+      [styles.invalid]:
+        typeof tone !== 'undefined' ? tone === TONE.CRITICAL : valid === false,
       [className]: className
     });
 
-    // eslint-disable-next-line react/prop-types
-    const { label, labelProps, invalid, help, helpProps, message, messageProps, secondaryLabel, tertiaryLabel, description } = this.props;
+    const {
+      // eslint-disable-next-line react/prop-types
```

Comment:
Calling this out: This duplication of `eslint-disable-next-line` is being used to maintain previous behaviour.
Future changes may want to understand why these PropTypes are not being generated.